webview/src/plots/components/App.test.tsx

```diff
@@ -1733,8 +1733,7 @@ describe('App', () => {
     it('should show a drop target at the end of the custom plots when moving a plot inside the section but not over any other plot', () => {
       renderAppWithOptionalDataInDragAndDropMode(
         {
-          custom: customPlotsFixture,
-          template: templatePlotsFixture
```

Comment:
[Q] From the test title it sounds like these plots are needed. Do the statements need to be updated?

Reply:
It was never testing the other part. I guess we need to update the statement.